site/en/docs/extensions/mv3/manifest/homepage_url/index.md

```diff
@@ -2,14 +2,18 @@
 layout: "layouts/doc-post.njk"
 title: "Manifest - Homepage Url"
 date: 2013-05-12
-updated: 2018-04-26
+updated: 2023-01-04
 description: Reference documentation for the homepage_url property of manifest.json.
 ---
 
-The URL of the homepage for this extension. The extensions management page (chrome://extensions)
-will contain a link to this URL. This field is particularly useful if you [host the extension on
```

Comment:
Did MV3 introduce any changes around how an extension can be hosted/what the behaviour is in that case? Wondering if it makes sense to keep this line or not.

Reply:
Question for Ian. He has the context on this PR.

Reply:
@oliverdunk My understanding is that this key only regards linking to the developer's home site, and not necessarily any explicit hosting of the extension.

Reply:
Let's follow up on this if anything, I don't think it needs to block the PR 👍